kitsune/questions/events.py

```diff
@@ -84,10 +82,7 @@ def _make_mail(locale, user, context):
             from kitsune.users.templatetags.jinja_helpers import display_name
 
             is_asker = asker_id == user.id
-            extra_params = {}
             if is_asker:
-                auth_str = get_auth_str(user)
```

Comment:
This is a important feature actually. I do not know if the feature should be removed.

The user should be logged in automatically when he clicks the link. I do not know if there are any issue to remove this feature.

Reply:
This will be tuned a bit to allow users to be redirected back after logging in if the don't have an active session. I was reluctant too to remove this but at the same time there is a security concern on how this is handled. Notification will still be there but users will have to login if they are not already logged in. Thoughts?